community/sway/etc/skel/.config/manjaro/manjaro-settings-manager.conf

```diff
@@ -0,0 +1,6 @@
+[mainwindow]
```

Comment:
no idea what that does, stolen from https://gitlab.manjaro.org/manjaro-arm/applications/arm-profiles/-/tree/master/overlays/sway/etc/skel/.config